examples/example.rs

```diff
@@ -11,7 +23,51 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             .unwrap()
     ));
 
+    let json = serde_json::json!({"foo": "bar"});
+    client.publish("demonstration", Cow::Owned(json)).await?;
     client.record_success().await?;
 
     Ok(())
 }
+
+async fn publish_subscribe(
```

Comment:
Thank you for providing an example along with your patch. This is great engineering.

Given that the old `fn example` is quite simple, I would be fine for `fn publish_subscribe` to replace it. Just an idea to reduce complexity. Not a strong opinion.

Reply:
Thanks for the feedback @mxinden!
If you don't mind I would keep it, not as test cases, I agree it's not really useful, but as an example for future users. I like the idea of having a really bare minimum "here is how you write a test that passes in testground", before moving into more complex synchronization patterns.